### DISCUSSION TITLE
I don't understand this security warning message from github

### DISCUSSION THREAD

**Discussion by bulgin** [community/community#37825] [Other Feature Feedback, Questions, & Ideas]
I recently received the following cryptic (to me, anyway) message.  I have no idea what this means.  What is read:org and what is user:email scopes.  This is above my pay grade.

A third-party OAuth application (Authentication Gateway) with read:org and user:email scopes was recently authorized to access your account.

**Comment by airtower-luna** [comment]
It means you allowed something (called "Authentication Gateway") access to certain things (the "scopes" say which) on your account. The warning is to make sure you're aware of this, and can revoke the access if you didn't actually mean to grant that access. Phishing that's trying to trick people into allowing access to private things is a thing, unfortunately, and so are mistakes.

I don't know what the "Authentication Gateway" is, is there anything you meant to give access to?

In general, check [Scopes for OAuth Apps](https://docs.github.com/en/developers/apps/building-oauth-apps/scopes-for-oauth-apps) to see what scopes mean. In your case:

> user:email | Grants read access to a user's email addresses.
> read:org | Read-only access to organization membership, organization projects, and team membership.
